### PR TITLE
Adding rolegal model to the spaCy universe

### PR DESCRIPTION
**Pull request by senisioi**
Adding a model for processing noisy Romanian legal documents.

## Description
Created a new entry in the spacy universe json to add the rolegal package. The model is trained using floret embeddings and predicts legal entity types.

### Types of change
Extending the Universe page.

## Checklist
- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by adrianeboyd** [comment]
Thanks for this contribution, it's especially nice to see floret vectors in use!

**Comment by senisioi** [comment]
thanks for the json fix :)

**Comment by adrianeboyd** [comment]
No worries, the format is a bit of a pain to write.

**Comment by adrianeboyd** [comment]
One more thing, in the preview the logo ends up rendered a bit strangely, maybe just because of the size?

https://deploy-preview-13017--spacy.netlify.app/universe/project/rolegal

Let me see if the raw URL works better...

**Comment by adrianeboyd** [comment]
Nope, still not recognized somehow...

Screenshot_2023-09-26_09-19-39

**Comment by senisioi** [comment]
made a smaller version, hope this one renders well

**Comment by adrianeboyd** [comment]
Sorry that wasn't as easy as it should have been! The preview looks good, I'll just make one more change to use "spaCy".

File: website/meta/universe.json [hunk 1]
```diff
@@ -4469,6 +4469,37 @@
             },
             "category": ["pipeline", "standalone"],
             "tags": ["spans", "rules", "ner"]
+        },
+        {
+            "id": "rolegal",
+            "title": "A Spacy Package for Romanian Legal Document Processing",
+            "thumb": "https://raw.githubusercontent.com/senisioi/rolegal/main/img/paper200x200.jpeg",
+            "slogan": "rolegal: a Spacy Package for Noisy Romanian Legal Document Processing",
+            "description": "This is a spacy language model for Romanian legal domain trained with floret 4-gram to 5-gram embeddings and `LEGAL` entity recognition. Useful for processing OCR-resulted noisy legal documents.",
```
Comment (adrianeboyd) [inline website/meta/universe.json:4478]:
```suggestion
            "title": "A spaCy Package for Romanian Legal Document Processing",
            "thumb": "https://raw.githubusercontent.com/senisioi/rolegal/main/img/paper200x200.jpeg",
            "slogan": "rolegal: a spaCy Package for Noisy Romanian Legal Document Processing",
            "description": "This is a spaCy language model for Romanian legal domain trained with floret 4-gram to 5-gram embeddings and `LEGAL` entity recognition. Useful for processing OCR-resulted noisy legal documents.",
```

**Comment by senisioi** [comment]
oh, indeed, thank you so much for making these changes for me :)

**Pull request merged**